dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/ApplicationProcessor.java

```diff
@@ -74,6 +78,13 @@
 
     private static final TraceComponent tc = Tr.register(ApplicationProcessor.class);
 
+    private static final PrivilegedAction<ThreadContextAccessor> getThreadContextAccessorAction = new PrivilegedAction<ThreadContextAccessor>() {
```

Comment:
If this is a constant, should it be named with CAPS_AND_UNDERSCORES?

Reply:
Also, why do we declare and use this instead of using `com.ibm.ws.util.ThreadContextAccessor.getPrivilegedAction()`?